app/views/notes/_responses.html.erb

```diff
@@ -5,7 +5,10 @@
     <% @responses = DrupalTag.find_nodes_by_type("#{response_type}:#{@node.id}", "note", 200) %>
     <% @unpaginated = true %>
  
-    <p id="<%= response_type %>s"><a class='btn btn-primary btn-lg' href="/post?tags=<%= response_type %>:<%= @node.id %><% if response_type == 'response' %>,hidden:response<% end %><%= ',' + (@tagnames.uniq.delete_if{|x| x.match(":") }).join(',') if @tagnames && @tagnames.length > 0 %>"><%= t('notes._responses.post_' + response_type) %></a></p>
+    <p id="<%= response_type %>s"><a class='btn btn-primary btn-lg' href="/post?tags=<%= response_type %>:<%= @node.id %><% if response_type == 'response' %>,hidden:response<% end %><%= ',' + (@tagnames.uniq.delete_if{|x| x.match(":") }).join(',') if @tagnames && @tagnames.length > 0 %>"><%= t('notes._responses.post_' + response_type) %></a>&nbsp;
+    <% if @node.power_tag('activity') %>
+    	<% if DrupalTag.where(name: @node.power_tag('activity')) %>
+    		<% @responses = DrupalTag.where(name: @node.power_tag('activity')).tag.nodes.first %><% end %><% end %></p>
```

Comment:
And see how we ask `@node.power_tag('activity')` three times? We could make that a bit more efficient too -- although it will probably cache the database call that fetches that info, so it's not too bad.